src/platform/ESP32/ConfigurationManagerImpl.h

```diff
@@ -35,6 +35,7 @@
 #include <platform/internal/GenericConnectivityManagerImpl_NoBLE.h>
 #endif
 
+#include "freertos/timers.h" // TimerHandle_t
```

Comment:
@feasel0 Why is this required here? Any compilation errors you see without this?

Reply:
The error (undefined reference to `TimerHandle_t`) seems to have gone away after the recent version update.  I've removed this line, since it's no longer needed.